app/components/DataTable/index.tsx

```diff
@@ -16,10 +16,13 @@ import GetAppIcon from '@material-ui/icons/GetApp';
 
 import ColumnPicker from './components/ColumnPicker';
 import LinkCellRenderer from './components/LinkCellRenderer';
+// not ideal, but no errors are thrown and no render loops occur
+// eslint-disable-next-line import/no-cycle
 import GeneCellRenderer from './components/GeneCellRenderer';
+// eslint-disable-next-line import/no-cycle
 import ActionCellRenderer from './components/ActionCellRenderer';
 import { getDate } from '../../utils/date';
-import ReportContext from '../../components/ReportContext';
+import ReportContext from '../ReportContext';
```

Comment:
to be consistent, might be worth using the `@` shorthand here despite the fact that its longer just so that you always import from `@/components` the same way when its outside the current component